<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
    background-image: url(imagenes/from-archivos.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover;
}
QLabel {
    background: none;  /* Asegúrate de que los QLabel no tengan fondo */
}
QPushButton {
    background: none;  /* Asegúrate de que los QPushButton no tengan fondo */
}
</string>
  </property>
  <widget class="QLineEdit" name="line_ruta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: transparent;
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_agregar">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="lista_archivos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: transparent;
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_ir_archivo">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_seleccionar_archivo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_eliminar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
    background-image: url(/home/pepo/PROYECTO FINAL/imagenes/secundaria.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover;
}
QLabel {
    background: none;  /* Asegúrate de que los QLabel no tengan fondo */
}
QPushButton {
    background: none;  /* Asegúrate de que los QPushButton no tengan fondo */
}


</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../imagenes/from-archivos-original.jpg</pixmap>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>line_ruta</zorder>
  <zorder>boton_agregar</zorder>
  <zorder>lista_archivos</zorder>
  <zorder>boton_ir_archivo</zorder>
  <zorder>boton_seleccionar_archivo</zorder>
  <zorder>boton_eliminar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
